src/Logary.sln

```diff
@@ -26,103 +26,103 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "core", "core", "{EA09AD42-B
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "integration-tests", "integration-tests", "{4755795D-4488-4013-81FC-9FD5D442B211}"
 EndProject
-Project("{f2a71f9b-5d33-465a-a702-920d77279786}") = "Logary.Targets.Riemann", "targets\Logary.Targets.Riemann\Logary.Targets.Riemann.fsproj", "{CC87925C-E4F2-4386-A322-8BC088DE7B33}"
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Logary.Targets.Riemann", "targets\Logary.Targets.Riemann\Logary.Targets.Riemann.fsproj", "{CC87925C-E4F2-4386-A322-8BC088DE7B33}"
```

Comment:
Done by VS, I'll remove these when I'm done, otherwise VS will just keep doing it